src/test/java/core/basesyntax/service/CreateFruitImplTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CreateFruitImplTest {
+    @Test
+    public void createFruit() {
+        String expectedName = "apple";
+        int expectedAmount = 5;
+        String testedName = "apple";
+        String testedAmount = "5";
+        Fruit expectedFruit = new Fruit();
+        expectedFruit.setName(expectedName);
+        expectedFruit.setAmount(expectedAmount);
+        Fruit fruit = new CreateFruitImpl().createFruit(testedName, testedAmount);
```

Comment:
create field FruitCreator and initialize it inside of method with @BeforeClass

Reply:
:)